packages/sindarian-i18n-cli/src/key-differ.test.ts

```
@@ -0,0 +1,110 @@
+
+    expect(result.added).toEqual(['a.key'])
+    expect(result.removed).toHaveLength(0)
+  })
+
+  it('throws descriptive error on malformed JSON', async () => {
+    const localePath = join(tempDir, 'malformed.json')
+    await writeFile(localePath, '{not valid json')
+
+    const messages = makeMessages(['a.key'])
+    await expect(diffKeys(messages, localePath)).rejects.toThrow(
+      /Failed to parse locale file.*malformed\.json/
```

ESLint

[error] Move this regular expression to module scope to avoid re-compilation on every call.

(e18e/prefer-static-regex)